include/sddf/util/string.h

```diff
@@ -8,44 +8,34 @@
 
 #pragma once
 #include <stddef.h>
+#include <stdint.h>
 
 #ifndef __has_builtin
 #  define __has_builtin(x) 0
 #endif
 
 static inline void *sddf_memset(void *s, int c, size_t n)
 {
-#if __has_builtin(__builtin_memset)
-    return __builtin_memset(s, c, n);
-#else
```

Comment:
This prevents using the builtin version at high optimisation levels.  What's needed is a memset implementation that is only linked if needed.  For both GCC and CLANG,  `__builtin_memset` is converted to a plain `memset` call at `-O1` or `-Os`

Reply:
I'd suggest adding out-of-line efficient implementations in libsddf_util.a